[2024-02-29T08:01:33.889+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_api_dag.get_weather_data scheduled__2024-02-19T01:00:00+00:00 [queued]>
[2024-02-29T08:01:33.926+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_api_dag.get_weather_data scheduled__2024-02-19T01:00:00+00:00 [queued]>
[2024-02-29T08:01:33.927+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-02-29T08:01:34.006+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): get_weather_data> on 2024-02-19 01:00:00+00:00
[2024-02-29T08:01:34.040+0000] {standard_task_runner.py:60} INFO - Started process 3970 to run task
[2024-02-29T08:01:34.068+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_api_dag', 'get_weather_data', 'scheduled__2024-02-19T01:00:00+00:00', '--job-id', '559', '--raw', '--subdir', 'DAGS_FOLDER/weather_api_dag.py', '--cfg-path', '/tmp/tmpq7i2ejth']
[2024-02-29T08:01:34.083+0000] {standard_task_runner.py:88} INFO - Job 559: Subtask get_weather_data
[2024-02-29T08:01:34.289+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_api_dag.get_weather_data scheduled__2024-02-19T01:00:00+00:00 [running]> on host 70f5ffd40a68
[2024-02-29T08:01:34.600+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_api_dag' AIRFLOW_CTX_TASK_ID='get_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-02-19T01:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-19T01:00:00+00:00'
[2024-02-29T08:01:34.956+0000] {logging_mixin.py:188} INFO - https://api.openweathermap.org/data/2.5/weather?q=bangkok&appid=97b6cde8520eb9524c22297524e3d9cd&units=metric
[2024-02-29T08:01:34.956+0000] {logging_mixin.py:188} INFO - {'coord': {'lon': 100.5167, 'lat': 13.75}, 'weather': [{'id': 802, 'main': 'Clouds', 'description': 'scattered clouds', 'icon': '03d'}], 'base': 'stations', 'main': {'temp': 35.45, 'feels_like': 40.55, 'temp_min': 33.94, 'temp_max': 37.73, 'pressure': 1008, 'humidity': 47, 'sea_level': 1008, 'grnd_level': 1007}, 'visibility': 10000, 'wind': {'speed': 5.43, 'deg': 186, 'gust': 4.05}, 'clouds': {'all': 28}, 'dt': 1709193237, 'sys': {'type': 2, 'id': 2081950, 'country': 'TH', 'sunrise': 1709163298, 'sunset': 1709205978}, 'timezone': 25200, 'id': 1609350, 'name': 'Bangkok', 'cod': 200}
[2024-02-29T08:01:34.958+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-29T08:01:35.007+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_api_dag, task_id=get_weather_data, execution_date=20240219T010000, start_date=20240229T080133, end_date=20240229T080135
[2024-02-29T08:01:35.139+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-29T08:01:35.199+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
